Driver was not properly inheriting CSA from STA
vap onto AP vap(s). This caused AP CSA to be
incoherent to connected clients and caused them to
disconnected depending on how strict they were.

mac80211/ath9k was disconnecting when a child pod
it was connected to was following parent pod
between channels 1 <-> 11 with:

> wlp1s0: AP 52:b4:f7:03:8f:19 switches to unsupported channel (2462 MHz, width:2, CF1/2: 2472/0 MHz), disconnecting
--- a/umac/mlme/ieee80211_mgmt_sta.c
+++ b/umac/mlme/ieee80211_mgmt_sta.c
@@ -653,6 +653,8 @@ void ieee80211_recv_beacon_sta(struct ie
                     }
                     ic->ic_chanchange_channel = chan;
                     ic->ic_chanchange_chwidth = ieee80211_get_chan_width(chan);
+                    ic->ic_chanchange_secoffset = ieee80211_sec_chan_offset(chan);
+                    ic->ic_chanchange_chanflag = chan->ic_flags;
                     /* Update node channel setting */
                     ieee80211_node_set_chan(ni);
                     ic->ic_flags |= IEEE80211_F_CHANSWITCH ;
